langlib/lang.map/src/main/ballerina/map.bal

```diff
@@ -56,6 +75,11 @@ public isolated function iterator(map<Type> m) returns object {
 
 # Returns the member of a map with given key.
 #
+# ```ballerina
+# map<int> indices = {a: 1, b: 2, c: 3, d: 4};
+# int indexOfA = indices.get("a");
```

Comment:
```suggestion
# int valueOfA = indices.get("a");
```

Having index prefix is correct but at the same time it's kind of misleading for this example because the intention is to get the value from a key.

Reply:
Let's also use a variable for the key. We should then be able to use a generic name for the result of get. E.g., if we had a map of employees, the variable for the result of get could be `employee`.